src/db/mod.rs

```diff
@@ -328,9 +327,7 @@ impl ThreadedDatabase for Database {
             doc = merge_options(doc, create_collection_options);
         }
 
-        self.command(doc, CommandType::CreateCollection, None).map(
-            |_| (),
-        )
+        self.command(doc, CommandType::CreateCollection, None).map(drop)
```

Comment:
What exactly is `drop` in this instance? I'm having trouble finding where this is defined.

Reply:
It's just `std::mem::drop` from the prelude. It's a slightly less convoluted way of ignoring the value, it does the exact same thing as `|_| ()`.

Reply:
Would it still work to just do `let _ = self.command(...)`? I think that's a bit easier to read since it doesn't require the user to know about/understand the external function.

If not, I'd definitely prefer that it be imported/invoked as `mem::drop` rather than just `drop`.